### ISSUE TITLE
Get the Device-ID when on Agent

### ISSUE THREAD

**Issue by prononext** [Ylianst/MeshCentral#5202]
Dear MeshCentral Fans, 

## does anyone know how to get the device ID when I am on a agent (Windows + Unix)?

The node ID of the group/mesh I can get with "$filepathtomyagent -nodeid" but I need the deviceid.

Background is that I want to automate some actions and notifications via meshctrl and to directly communicate with the device from the agent or another host I need to get the deviceid. 

I am happy for any hint on this topic. 

Have a great day.

**Comment by si458** [comment]
i think you only need the nodeid to communicate to a computer over meshctrl?
what do you mean by deviceid exactually?

**Comment by prononext** [comment]
That is what I also thought but the nodeid is different from the deviceid which I use in the meshctrl command.

So I get the nodeid via my command above and then I compared it to the one which is in the URL of my meshcentral server inside that specific device.

I tried both ids with meshctrl, but only the one I got from meshcentral GUI worked with meshctrl and not the one I got with "meshagent -nodeid" on the agent itself.

Any ideas?
